test/scaling_test/scaling_test.go

```diff
@@ -52,15 +52,12 @@ var _ = Describe("Scaling test", func() {
 	Context("Standalone deployment (S1)", func() {
 		It("scaling_test: Can Scale Up and Scale Down Standalone CR", func() {
 
-			standalone, err := deployment.DeployStandalone(deployment.GetName())
+			standalone, err := deployment.DeployStandalone(deployment.GetName(), "", "")
 			Expect(err).To(Succeed(), "Unable to deploy standalone instance ")
 
 			// Wait for Standalone to be in READY status
 			testenv.StandaloneReady(deployment, deployment.GetName(), standalone, testenvInstance)
 
-			// Wait for Monitoring Console Pod to be in READY status
```

Comment:
We have removed the previously commented waits for MC to become ready. Is this because these scaling tests don't specifically add the MC CR, and instead the scaling scenario has been covered in the crud tests?